storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/StorIOSQLite.java

```diff
@@ -144,6 +145,110 @@
     }
 
     /**
+     * Allows observe changes of required tags.
+     * <p/>
+     * Tags are the addition level of notifications that provides
```

Comment:
`Tags are optional meta information that you can attach to Changes object to have more fine-grained control over observing changes in the database.`?